act-rules-format.bs

```diff
@@ -17,19 +17,19 @@ Introduction {#intro}
+This format is intended to enable a consistent interpretation of how to test for accessibility requirements so as to avoid conflicting results of accessibility tests. It is intended to be applicable to describe both manual accessibility tests as well as automated testing done through accessibility test tools (ATTs).
 
 Describing how to test certain accessibility requirements will result in accessibility tests that are transparent with test results that are reproducible. The Accessibility Conformance Testing (ACT) Rules Format defines the requirements of these test descriptions, known as Accessibility Conformance Testing Rules (ACT Rules).
 
 
 Scope {#scope}
 ==============
 
-The ACT Rules Format scope is focused on the development of ACT Rules for the conformance testing of content created using web technologies, such as HTML, CSS, WAI-ARIA, SVG and more, including digital publishing. The ACT Rules Format, however, is designed to be technology agnostic, meaning it has no specific technology in mind. This also means that the ACT Rules Format could conceivably be used for other technologies.
+The ACT Rules Format defined in this specification is focused on the description of test rules applicable to to content created using web technologies, such as HTML, CSS, WAI-ARIA, SVG and more, including digital publishing. The ACT Rules Format, however, is designed to be technology agnostic, meaning it has no specific technology in mind. This also means that the ACT Rules Format could conceivably be used for other technologies.
 
-Accessibility requirements such as Web Content Accessibility Guidelines, which is specifically designed for web content, can be tested using ACT Rules.
+For instance, the ACT Rules Format can be used to describe ACT Rules dedicated to testing the accessibility requirements defined in the Web Content Accessbility Guidelines, which are specifically designed for web content.
 
-Other accessibility requirements that may be applicable to web technologies should also be testable with ACT Rules. For example, the User Agent Accessibility Guidelines 2.0 is applicable to web-based user agents, for which ACT Rules could be developed, but other technologies can also be used to develop User Agents, which are not web-based. Because some accessibility requirements may be applicable to technologies other than web technologies, the ACT Rules Format may not be suitable for every part of these accessibility requirement.
+Other accessibility requirements that may be applicable to web technologies would also typically be testable with ACT Rules. For example, as the User Agent Accessibility Guidelines 2.0 is applicable to web-based user agents, ACT Rules could be developed that target these requirements. However, as other technologies which are not web-based can also be used to develop User Agents, the ACT Rules Format would not necessarily be suitable to every part of the User Agent Accessibility Guidelines 2.0.
```

Comment:
Other accessibility requirements applicable to web technologies could also be testable with ACT Rules. 
For example, ACT Rules could be developed to test the conformance of web-based user agents to the User Agent Accessibility Guidelines.
However, the ACT Rules Format would not necessarily be suitable to describe tests for the conformance of a non-web-based user agent.